src/components/files-upload-overview.yml

```diff
@@ -0,0 +1,10 @@
+headline:
+  en: Files upload
+  de: Dateien Hochladen
```

Comment:
"hochladen" should be lower cased

Reply:
I was intending for a noun there 🤔 Maybe we could say something like "Files upload overview", how would that be in German?

Reply:
Ahh, I get it. Then you can use the nice German noun "Datei-Upload" :laughing: